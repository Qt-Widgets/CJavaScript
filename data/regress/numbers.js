console.log(3);
console.log(-12);
console.log(1000000);

console.log(-0123);
console.log(0377);

console.log(0xff);
console.log(-0xCAFE911);

console.log(3.14);
console.log(-1.414);
console.log(.3333333333);
console.log(6.02e+23);
console.log(1.4738223E-32);
