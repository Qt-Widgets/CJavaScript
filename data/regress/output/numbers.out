3
-12
1e+06
-83
255
255
-2.12855e+08
3.14
-1.414
0.333333
6.02e+23
1.47382e-32
