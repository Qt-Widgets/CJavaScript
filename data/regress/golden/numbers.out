3
-12
1000000
-83
255
255
-212855057
3.14
-1.414
0.3333333333
6.02e+23
1.4738223e-32
